app/gcl/professional-cloud-network-engineer/section3-load-balancing/page.css

```
@@ -0,0 +1,613 @@
+    width: 100%;
+    display: flex;
+    justify-content: safe center;
+    overflow-x: auto;
+    margin: 32px 0;
+    background: var(--color-pcne-s3-elevated, #0d1a2c);
+    border: 1px solid var(--color-pcne-s3-border, #1c3350);
+    border-radius: 12px;
+    padding: 24px;
+}
+
+.pcne-s3-page .mermaid-wrap :global(svg) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)